<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>530</y>
      <width>75</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>退出
程序</string>
    </property>
   </widget>
   <widget class="QLabel" name="TitleLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>387</width>
      <height>59</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>style1</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>TitleLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>779</width>
      <height>59</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>style2</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectPLCBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>480</y>
      <width>75</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>连接PLC</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="exampleGrp">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>示例</string>
    </property>
    <widget class="QLabel" name="stopStripExamp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>style4</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="normalStripExamp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>style5</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="errStripExamp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>style6</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="stopTxt">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>style7</string>
     </property>
     <property name="text">
      <string>计划停止</string>
     </property>
    </widget>
    <widget class="QLabel" name="normalTxt">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>style7</string>
     </property>
     <property name="text">
      <string>正常</string>
     </property>
    </widget>
    <widget class="QLabel" name="errTxt">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>style7</string>
     </property>
     <property name="text">
      <string>异常</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="StateGrp">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>390</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>状态信息</string>
    </property>
    <widget class="QLabel" name="commStateTxt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>style8</string>
     </property>
     <property name="text">
      <string>通信状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="allErrTxt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>style8</string>
     </property>
     <property name="text">
      <string>综合异常</string>
     </property>
    </widget>
    <widget class="QLabel" name="lockStateTxt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>style8</string>
     </property>
     <property name="text">
      <string>互锁切</string>
     </property>
    </widget>
    <widget class="QLabel" name="PLCRunTxt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>style8</string>
     </property>
     <property name="text">
      <string>PLC运行</string>
     </property>
    </widget>
    <widget class="QLabel" name="CommState">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ErrState">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="LockState">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="PlcRunState">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="errTblWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>401</y>
      <width>256</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="errTblTitle">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>380</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>style9</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
